lib/ui/platform_dispatcher.dart

```diff
@@ -623,6 +623,19 @@ class PlatformDispatcher {
   @FfiNative<Void Function(Handle)>('PlatformConfigurationNativeApi::SetIsolateDebugName')
   external static void _setIsolateDebugName(String name);
 
+  /// Requests the Dart VM to adjusts the GC heuristics based on the requested `performance_mode`.
+  ///
+  /// Returns the old performance mode of the Dart VM.
+  ///
+  /// See [DartPerformanceMode] for more information on individual performance modes.
+  DartPerformanceMode requestDartPerformanceMode(DartPerformanceMode mode) {
```

Comment:
I would not necessarily return the current performance mode. What is the plan for this value?

Reply:
I was mostly going to use it when adding tests on the framework side, makes it easy to assert instead of having to add `GetCurrentPerformanceMode`. Let me know if you think this is a bad idea, happy to make it a void function.

Reply:
I would consider removing it, and the asserts in the framework should track whether or not the requested GC modes are consistent. 

Consider: we may wish to disable this functionality on certain platforms, or tune other changes in the engine that overrwrite the requested mode. If we tell people what the resulting mode is we're going to make that part of the contract of this API - which folks will depend on, eventually.

Also consider: in the event the framework is making bad requests, but the engine is ignoring those requests for an unrelated reason, we'd like the framework asset/test to still fail

Reply:
Those are valid points, will make it not return anything for now.